### ISSUE TITLE
The download link for ADE20K cannot be opened.

### ISSUE THREAD

**Issue by Liqq1** [open-mmlab/mmsegmentation#2869]
Hi 👋, the download link(as shown in below) for ADE20K cannot be opened
`ADE20K`
The training and validation set of ADE20K could be download from this [link](http://data.csail.mit.edu/places/ADEchallenge/ADEChallengeData2016.zip). We may also download test set from [here](http://data.csail.mit.edu/places/ADEchallenge/release_test.zip).
